.github/workflows/stale_bot.yml

```diff
@@ -9,8 +9,14 @@ on:
 #            │ │ │ │ ┌───────── day of the week (0 - 6 or SUN-SAT)
     - cron: '0 5 * * *' # Every day at 05:00 UTC
 
+permissions:
+  contents: read
+
 jobs:
   stale:
+    permissions:
+      issues: write  # for actions/stale to close stale issues
+      pull-requests: write  # for actions/stale to close stale PRs
```

Comment:
Thanks for the PR! To reduce the amount of workflow-level configuration I think we should instead [set the token to be read-only by default in the repository settings](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-the-default-github_token-permissions). That way we would only need to add these three lines. (We don't need to update the changelog workflow as it is disabled.)

Reply:
I have changed that to be read only on the settings page.

Reply:
Do you need anything from me? Thanks

Reply:
If you like you can edit the commit to only include these three lines, or we can update it?

Reply:
I can do that.

Reply:
Thanks!